### ISSUE TITLE
Block Listener Attempts Database Queries, Before Zikula is Installed

### ISSUE THREAD

**Issue by shefik** [zikula/core#2764]
Under Zikula 1.4.2 build 103, this error is thrown instead:

```
Oops! An Error Occurred

The server returned a "500 Internal Server Error".

Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

An exception occurred while executing 'SELECT t0.id AS id_1, t0.modname AS modname_2, t0.name AS name_3, t0.value AS value_4 FROM module_vars t0': SQLSTATE[3D000]: Invalid catalog name: 1046 No database selected
```

The problem is due to the Block listener attempting to make database queries, before Zikula is installed.

**Comment by craigh** [comment]
can you post the full trace?

**Comment by craigh** [comment]
oh - fixed by #2765
